src/confetti.js

```diff
@@ -257,13 +258,21 @@
     document.body.appendChild(canvas);
 
     animationObj = animate(canvas, fettis, function () {
-      animationObj = null;
-      document.body.removeChild(canvas);
+      destroy();
     });
 
     return animationObj.promise;
   }
 
+  function destroy() {
```

Comment:
So there are a couple of issues with this new method:

1. It doesn't actually stop the animation or any of the work that the module is doing. It will continue to do a lot of calculations at the moment, even after `destroy` is called (until it is done, of course). It might even though because `canvas` is now null yet it is needed by the rest of the code.
2. It does not do anything with the promise that is returned. The promise is neither resolved nor rejected, and that feels wrong. I am actually not sure which one should happen, but there should definitely be no more code executing after a `destroy` method, so the promises need to complete in one way or another during this method.
3. There are no automated tests added. Public methods (as well as new functionality) really should be tested.